pkg/cloud/network_test.go

```diff
@@ -99,6 +99,15 @@ var _ = Describe("Network", func() {
 		It("correctly identifies an existing network from a network status", func() {
 			Ω(cloud.NetworkExists(dummies.CSCluster.Status.Zones.GetOne().Network)).Should(BeTrue())
 		})
+
+		// It("resolves network details with network ID instead of network name", func() {
```

Comment:
Why is this test case commented?

Reply:
This test is covered in other tests now, I just didn't get around to removing it.

Removed. : )